components/greaselion/browser/greaselion_download_service.cc

```diff
@@ -38,12 +39,15 @@ const char kURLs[] = "urls";
 const char kScripts[] = "scripts";
 const char kRunAt[] = "run_at";
 const char kMessages[] = "messages";
+const char kMinimumBraveVersion[] = "minimum_brave_version";
 // precondition keys
 const char kRewards[] = "rewards-enabled";
 const char kTwitterTips[] = "twitter-tips-enabled";
 const char kRedditTips[] = "reddit-tips-enabled";
 const char kGithubTips[] = "github-tips-enabled";
 const char kAutoContribution[] = "auto-contribution-enabled";
+const char kSupportsMinimumBrowserVersion[] =
+    "supports-minimum-browser-version";
```

Comment:
Should this be `kSupportsMinimumBraveVersion` to be consistent with `kMinimumBraveVersion` (or vice versa)?

Reply:
I'm thinking vice versa